client/src/legacy/SiteConfig.js

```diff
@@ -1,45 +1,14 @@
-/* global window */
-const hiddenClass = 'mfa-settings--hidden';
```

Comment:
Is there some CSS to remove here too?

Reply:
I believe I've nabbed it already (in `SiteConfigExtension.php`), but can search to be sure that's all of it :)

Reply:
See SiteConfig.scss in the legacy folder

Reply:
Ah, you meant the scss rule :joy: